cirq/circuits/circuit_test.py

```diff
@@ -3375,25 +3455,25 @@ def test_insert_operations_errors():
 
 
 def test_validates_while_editing():
-    c = cirq.Circuit(device=cg.Foxtail)
+    c = cirq.Circuit(device=foxy)
 
     with pytest.raises(ValueError, match='Unsupported qubit type'):
         # Wrong type of qubit.
         c.append(cirq.Z(cirq.NamedQubit('q')))
     with pytest.raises(ValueError, match='Qubit not on device'):
         # A qubit that's not on the device.
         c[:] = [cirq.Moment([cirq.Z(cirq.GridQubit(-5, 100))])]
-    c.append(cirq.Z(cirq.GridQubit(0, 0)))
+    c.append(cirq.Z(cirq.GridQubit(0, 1)))
```

Comment:
Why this change?

Reply:
Otherwise there was an overlapping moment exception. I added the logic now to the testclass in `can_add_operation_into_moment` to call `super()` like Foxtail does, so now this change can be reverted.